definitions/json/AuthorisationCaseField.json

```diff
@@ -18109,6 +18109,90 @@
     "UserRole": "et-acas-api",
     "CRUD": "R"
   },
+  {
```

Comment:
These new fields were added in PET-1.1.1.3 release. I think these should go into master on next release